apps/web/src/app/(non-evm)/aptos/pool/[address]/page.tsx

```diff
@@ -62,24 +60,23 @@ const Pool = () => {
       <div className="flex flex-col gap-6">
         <div className="grid grid-cols-1 md:grid-cols-[auto_400px] gap-6">
           <div className="flex flex-col gap-6">
-            <ManageV2LiquidityCard />
-            {pool?.id ? <PoolComposition row={pool} /> : null}
+            <ManageV2LiquidityCard address={address} />
+            <PoolComposition row={pool} />
```

Comment:
might be nice to have loading states for these things but would be a lot of work for not a lot of gain (#TODO #FixSomeday)